autopilot/contractor.go

```diff
@@ -470,11 +473,6 @@ func (c *contractor) runContractRenewals(ctx context.Context, cfg api.AutopilotC
 	for _, ci := range toRenew {
 		// TODO: keep track of consecutive failures and break at some point
 
-		// break if the contractor was stopped
-		if c.isStopped() {
```

Comment:
same question

Reply:
resolving, duplicate of https://github.com/SiaFoundation/renterd/pull/129/files#r1098891311 tackling it there